### PR TITLE
fix: add pod yaml

### PR DESCRIPTION
**Pull request by d2lam**
## Context
We want to allow builds to run inside a VM in Kubernetes. 
This pod yaml will pass arguments into `/sd/hyper-runner.sh`. This script is available in the  `{{base_image}}`, and is expected to start the VMs
`baseImage` is configurable by cluster manager via an environment variable: https://github.com/screwdriver-cd/screwdriver/pull/625/files

## Objective
To allow build to run inside a VM in Kubernetes

## References
Related to https://github.com/screwdriver-cd/screwdriver/issues/597

### CODE REVIEW

File: config/pod.yaml.tim [hunk 1]
```diff
@@ -4,52 +4,69 @@ metadata:
...
+      privileged: true
     resources:
       limits:
         memory: 2Gi
-    command:
-    - "/opt/sd/tini"
-    - "--"
-    - "/bin/sh"
-    - "-c"
-    # Run the launcher and logservice in the background
```
Comment (ian-fox) [inline config/pod.yaml.tim]: The hyper-runner.sh does this part now, right? Is there anywhere I can see that? Looking at https://github.com/screwdriver-cd/screwdriver/pull/625/files I see that it *can* be configured, but not what the default is.

File: config/pod.yaml.tim [hunk 2]
```diff
@@ -4,52 +4,69 @@ metadata:
...
+         {
+           "name": "sdlauncher",
+           "mountPath": "/opt/launcher"
+         }
+       ]
+     }
+  ]'
 spec:
-  serviceAccount: {{service_account}}
+  tolerations:
```
Comment (minz1027) [inline config/pod.yaml.tim]: No need for this

File: config/pod.yaml.tim [hunk 3]
```diff
@@ -4,52 +4,69 @@ metadata:
...
+    - key: vm
+      value: "true"
+      effect: NoSchedule
+  nodeSelector:
+    vm: hyperd
   restartPolicy: Never
   containers:
-  - name: build
-    image: {{container}}
+  - name: docker
```
Comment (minz1027) [inline config/pod.yaml.tim]: rename it

File: config/pod.yaml.tim [hunk 4]
```diff
@@ -4,52 +4,69 @@ metadata:
...
     volumeMounts:
-    - mountPath: /opt/sd
-      name: screwdriver
-    - mountPath: /sd
-      name: workspace
-    env:
-    - name: SD_TOKEN
-      value: "{{token}}"
+    - mountPath: /var/run
+      name: var-run-host
```
Comment (minz1027) [inline config/pod.yaml.tim]: hyper socket

**Comment by dchinmaya** [comment]
👍 once the review comments are fixed

File: config/pod.yaml.tim (hunk 1 above)
Comment (d2lam) [inline config/pod.yaml.tim]: yup

**Pull request merged**